package.json

```diff
@@ -51,12 +51,15 @@
     "@addepar/sass-lint-config": "^2.0.1",
     "@addepar/style-toolbox": "~0.8.1",
     "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.0.0",
     "@ember/test-helpers": "^2.8.1",
+    "@embroider/test-setup": "^4.0.0",
+    "@glimmer/component": "^1.1.2",
```

Comment:
embroider complains if this isn't present because the dummy app imports from it here: https://github.com/Addepar/ember-table/blob/3bb089665b2031e74c86bf9b5691976f9249ef18/tests/dummy/app/components/examples/aborting-a-selection/component.js#L1